tests/unit/dags/conftest.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+
+import pytest
+from airflow.models import DagBag
+
+
+@pytest.fixture()
+def dagbag(provider_dir):
+    return DagBag(dag_folder=f"{str(provider_dir)}/example_dags", include_examples=False)
+
+
+def assert_dag_dict_equal(source, dag):
+    assert dag.task_dict.keys() == source.keys()
+    for task_id, downstream_list in source.items():
+        assert dag.has_task(task_id)
+        task = dag.get_task(task_id)
+        assert task.downstream_task_ids == set(downstream_list)
```

Comment:
The common way to have these helper functions is either use the pytest-plugin: https://pypi.org/project/pytest-helpers-namespace/

or wrap this code in a class (as a static method), e.g.
```python
class _DagAssertions:
    @staticmethod
    def assert_dict_equal(source, dag):
        assert dag.task_dict.keys() == source.keys()
        for task_id, downstream_list in source.items():
            assert dag.has_task(task_id)
            task = dag.get_task(task_id)
            assert task.downstream_task_ids == set(downstream_list)

@pytest.fixture
def DagAssertions():
    return _DagAssertions
```